server/fleet/mdm.go

```diff
@@ -1042,15 +1042,20 @@ const (
 	IPadOS
 )
 
-type AppleDevicePlatform string
+type InstallableDevicePlatform string
 
 const (
-	MacOSPlatform  AppleDevicePlatform = "darwin"
-	IOSPlatform    AppleDevicePlatform = "ios"
-	IPadOSPlatform AppleDevicePlatform = "ipados"
+	MacOSPlatform   InstallableDevicePlatform = "darwin"
+	IOSPlatform     InstallableDevicePlatform = "ios"
+	IPadOSPlatform  InstallableDevicePlatform = "ipados"
+	AndroidPlatform InstallableDevicePlatform = "android"
 )
 
-var VPPAppsPlatforms = []AppleDevicePlatform{IOSPlatform, IPadOSPlatform, MacOSPlatform}
+func (p InstallableDevicePlatform) IsValidInstallableDevicePlatform() bool {
+	return p == IOSPlatform || p == IPadOSPlatform || p == MacOSPlatform || p == AndroidPlatform
```

Comment:
You can use `slices.Contains(VPPAppsPlatforms, p)` instead, so the list of platforms is maintained in a single spot.